File: backend/DAL/Entities/DangerResolveRequest.cs
namespace DAL.Entities;

public class DangerResolveRequest : Entity {
  public DateTime Timestamp { get; set; }
  public Guid UserId { get; set; }
  public Guid DangerId { get; set; }

  // Foreign Keys
  public User User { get; set; }
  public Danger Danger { get; set; }
}

File: frontend/src/services/entities/request/DangerResolveRequest.ts
import { Danger } from "../Danger"
import { User } from "../User"

export interface DangerResolveRequest {
  id?: string
  createdAt?: Date
  updatedAt?: Date | undefined
  timestamp?: Date
  userId?: string
  dangerId?: string
  user?: User
  danger?: Danger
}
